src/coll/mod.rs

```diff
@@ -579,6 +584,26 @@ impl Collection {
             .await?;
         Ok(())
     }
+
+    /// Starts a new `ChangeStream` that receives events for all changes in this collection.
```

Comment:
Can we update this `ChangeStream` to be a link? Check out the docstring above for the `Collection` struct for an example of that.

Ditto for other places we refer to our types (e.g. `ChangeStream` or `ResumeToken`) from files other than the ones they're defined in.

Reply:
fixed